frontend/src/index.css

```diff
@@ -38,6 +38,19 @@ label {
     color: #ffdbec;
 }
 
+a:link {
+    color: #F251B1;
+}
+
+a:visited {
+    color: #BD86FF;
+}
+
+a:hover {
```

Comment:
```suggestion
a:link:hover {
```
This should be `a:link:hover `instead of `a:hover`, so that the hover effect applies only to links. This is because (strangely) dangerouslySetInnerHTML wraps text content after links in` <a> `even though they're not links.
(What happens as is:)
![image](https://user-images.githubusercontent.com/31525446/130287349-aefa2c87-7cd2-463c-9f66-1a64b0435f84.png)


Make sure you also change `a:visited:hover` to a suitable color style.
```
a:visited:hover {
    background-color: #F251B1;
    color: #ffdbec;
}
``` 
although maybe you'd want to change the hover style a bit to distinguish between links visited and not visited.